setup.cfg

```diff
@@ -0,0 +1,41 @@
+[metadata]
+name = woeusb-ng
+version = 0.2.10
+author = Jakub Szymański
+author_email = jakubmateusz@poczta.onet.pl
+description = woeusb-ng is a simple tool that enable you to create your own usb stick windows installer from an iso image or a real DVD. This is a rewrite of original WoeUSB.
```

Comment:
Please preserve the casing of the product name.

Reply:
I removed the capitalization because it conflicts with debian's packaging guide. Should I restore them?

Reply:
Could you mention which section of the debian packaging guide/debian policy requires that lower case is required in the package description?

Reply:
Sorry. My bad. It's the package names. Not in the description.. I'll change the descriptions. Thanks for the clarification.